content/docs/command-reference/commit.md

```diff
@@ -17,65 +17,52 @@ positional arguments:
 
 ## Description
 
-The `dvc commit` command is useful for several scenarios, when data already
-tracked by DVC changes: when a [stage](/doc/command-reference/run) or
-[pipeline](/doc/command-reference/dag) is in development/experimentation; to
-force-update the `dvc.lock` or `.dvc` files without reproducing stages or
```

Comment:
> Are there any significant important fixes/changes here?

Yes @shcheklein this became a full revision... I'll extract this to its own PR at this point ⌛

Reply:
Moved to https://github.com/iterative/dvc.org/pull/1989#pullrequestreview-541500924